XTS/WS-C/dev/src/com/arjuna/webservices11/wscoor/client/WSCOORClient.java

```diff
@@ -41,12 +42,27 @@
     private static synchronized ActivationService getActivationService()
     {
         if (activationService.get() == null) {
-            // activationService.set(new ActivationService(null, new QName("http://docs.oasis-open.org/ws-tx/wscoor/2006/06", "ActivationService")));
-            activationService.set(new ActivationService());
+            setActivationService();
```

Comment:
I saw it in WildFly as well as in other places. I think in general it is preferred to have small methods with single task.
Why do you think it's bad to split them up?